test/Microsoft.AspNet.Routing.Tests/RouteTest.cs

```diff
@@ -631,6 +631,60 @@ public void GetVirtualPath_Fail()
             Assert.Null(path);
         }
 
+        [Fact]
+        public void GetVirtualPath_ForList()
```

Comment:
Could you change this test to be `GetVirtualPath_ForListOfStrings` and add one more test for `GetVirtualPath_ForListOfInts` to show that string conversion happens for non-strings?